rollup-http/rollup-http-server/src/http_service.rs

```diff
@@ -148,6 +150,25 @@ async fn report(report: Json<Report>, data: Data<Mutex<Context>>) -> HttpRespons
     };
 }
 
+/// Process gio request and return the result
+#[actix_web::post("/gio")]
+async fn gio(request: Json<GIORequest>, data: Data<Mutex<Context>>) -> HttpResponse {
+    log::debug!("received gio request {:#?}", request);
+    let context = data.lock().await;
+    // Write report to linux rollup device
```

Comment:
I recognize I'm being pedantic with this comment, but here it goes:
This comment got inserted here and removed on the next commit that states it is only formatting the code.

Formatting first and then implementing a feature (while keeping the formatting) makes the review process easier, consider that for next time.